assertion_jwt_grant_type.go

```diff
@@ -0,0 +1,92 @@
+/*
+Copyright 2015 Rodrigo Rafael Monti Kochenburger
```

Comment:
These are out of date and should probably be for "Doximity, Inc." Can we rely on the `LICENSE` file in the root to handle licensing of files and remove this comment?